Gruntfile.js

```diff
@@ -1,134 +1,149 @@
+    'style',
+    // Extra examples:
+    // 'component/admin-tabs',
+    // 'component/02-molecule/site-search'
+    // 'layout/article'
+  ];
+
+  /**
+   * Dynamically builds list of files for grunt config
+   */
+  const scssFiles = {};
+  const pxToRemCssFiles = {};
+  styleFiles.forEach((filePath) => {
+    scssFiles[`css/${filePath}.css`] = `scss/${filePath}.scss`;
+    pxToRemCssFiles[`css/${filePath}.css`] = [`css/${filePath}.css`,];
+  });
 
   grunt.initConfig({
-    pkg: grunt.file.readJSON('package.json'),
-    sass: {
-      options: {
-        // sourceMap: true
+    'pkg': grunt.file.readJSON('package.json'),
+    'sass': {
+      'options': {
+        'sourceMap': isDev,
+      },
+      'dist': {
+        'files': scssFiles,
       },
-      dist: {
-        files: {
-          'css/style.css': 'scss/style.scss'
-        }
-      }
     },
-    sass_globbing: {
-      dist: {
-        files: {
-          'scss/_component.scss': 'scss/component/**/*.scss',
-          'scss/_layout.scss': 'scss/layout/**/*.scss',
-          'scss/_skin.scss': 'scss/skin/**/*.scss'
-        }
+    'sass_globbing': {
+      'dist': {
+        'files': {
+          'scss/_component.scss': 'scss/component/**/_*.scss',
+          'scss/_layout.scss': 'scss/layout/**/_*.scss',
+          'scss/_skin.scss': 'scss/skin/**/_*.scss',
+        },
+        'options': {
+          'signature': '// generated with grunt-sass-globbing \n\n',
+        },
+      },
+      'options': {
+        'useSingleQuotes': true,
       },
-      options: {
-        useSingleQuotes: true
-      }
     },
-    watch: {
-      css: {
-        files: ['scss/**/*.scss'],
-        tasks: ['sasslint', 'sass_globbing', 'sass', 'autoprefixer'],
-        options: {
-          livereload: true,
-          spawn: false
-        }
-      }
+    'postcss': {
+      'options': {
+        'map': isDev ?
+          {
+            'inline': false,
+            'annotation': 'css/postcss/maps/',
+          }
+          : false,
+        'processors': !isDev ?
+          [
+            // Add vendor prefixes
+            require('autoprefixer')({'browsers': '> 0.25%, last 2 versions, Firefox ESR, not dead',}),
+            // Minify the result
+            require('cssnano')(),
+          ] : [
+            // Add vendor prefixes
+            require('autoprefixer')({'browsers': '> 0.25%, last 2 versions, Firefox ESR, not dead',}),
+          ]
+        ,
+      },
+      'dist': {
+        'src': 'css/**/*.css',
+      },
     },
-    clean: {
-      css: ['css/style.css', 'scss/_component.scss', 'scss/_layout.scss', 'scss/_skin.scss']
+    // PostCSS plugin for px to rem isn't working, this is...
+    'px_to_rem': {
+      'dist': {
+        'options': {
+          'base': 16,
+          'fallback': false,
+          'fallback_existing_rem': false,
+          'ignore': [],
+          'map': isDev,
+        },
+        'files': pxToRemCssFiles,
+      },
     },
-    wiredep: {
```

Comment:
I think we might want to remove Bower and Wiredep for a couple of reasons:
* Wiredep is adding 3rd party code to one D8 library, which doesn't feel like a wise idea. I don't see this as a big time saver and it's encouraging a bad practice from what I can tell.
* With D8's libraries we have a standardized way to add metadata like what version the library is, and it's a few less build steps. There are a few drawbacks though:
  * Committing 3rd party code to the repo (which we don't usually have too many 3rd party libraries on most of our sites)
  * Relying on peer reviews and humans to get that metadata in

I think it's worth it, but thought I'd point out the change.